guides/common/modules/proc_adding-azure-details-to-a-compute-profile.adoc

```diff
@@ -26,8 +25,16 @@ Note that the username that you enter for compute profile must be the same that
 . Optional: In the *Custom Script Command* field, enter commands to perform on the virtual machine when the virtual machine is provisioned.
 . Optional: If you want to run custom scripts when provisioning finishes, in the *Comma separated file URIs* field, enter comma-separated file URIs of scripts to use.
 The scripts must contain `sudo` at the beginning because {ProjectName} downloads files to the `/var/lib/waagent/custom-script/download/0/` directory on the host and scripts require sudo privileges to be executed.
+. If you want to add an *NVIDIA Driver* you can as well by selecting the *NVIDIA driver / CUDA* checkbox. 
```

Comment:
```suggestion
. (Optional) Add an *NVIDIA Driver* by selecting the *NVIDIA driver / CUDA* checkbox. 
```

Reply:
`Optional:` would be in-line with IBM Style

Reply:
my bad!